forge-game/src/main/java/forge/game/spellability/AbilityManaPart.java

```diff
@@ -387,8 +387,9 @@ public boolean meetsManaRestrictions(final SpellAbility sa, String restrictions)
                 continue;
             }
 
-            if (restriction.equals("Disturb")) {
-                if (sa.isDisturb()) {
+            if (restriction.equals("FaceDownOrTurnFaceUp")) {
+                if ((sa.isSpell() && sa.getHostCard().isCreature() && sa.isCastFaceDown())
```

Comment:
imo no creature check here?
e. g. _Lumithread Field_

Reply:
You are wrong

It's about the Face Down Creature Spell

Reply:
But technically, all face down spells are creature spells

Reply:
Right, but somehow the check fails for me there
> Spend this mana only to cast face-down spells or to turn creatures face up.

Reply:
The check can probably be removed 

Below too

Because like I said all facedown spells should be creatures.
But otherwise we might need to debug it. I think the Spell Host might not be set yet

Reply:
fixed

Reply:
i asked Dunkatog about the different wording:

* technically there is a difference between "face down spells" and "face down creature spells"
* but in practice, all face down spells are also creature spells 

but they aren't going to update the other card